domains/measurements/Measurements.API/API/Controllers/MeasurementsController.cs

```diff
@@ -1,54 +1,60 @@
+using System.Net.Http.Headers;
 using API.Models;
 using API.Models.Request;
 using API.Services;
-using EnergyOriginAuthorization;
 using FluentValidation;
 using FluentValidation.AspNetCore;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace API.Controllers;
 
 [ApiController]
 [Authorize]
-public class MeasurementsController : AuthorizationController
+public class MeasurementsController : ControllerBase
 {
+
     [HttpGet]
     [Route("api/measurements/consumption")]
-    public async Task<ActionResult<MeasurementResponse>> GetConsumptionMeasurements([FromQuery] MeasurementsRequest request, IMeasurementsService measurementsService, IValidator<MeasurementsRequest> validator)
+    public async Task<ActionResult<MeasurementResponse>> GetConsumptionMeasurements([FromQuery] MeasurementsRequest request, IMeasurementsService measurementsService, IValidator<MeasurementsRequest> validator, [FromServices] IHttpContextAccessor httpContextAccessor)
```

Comment:
Did you test that it works?

Reply:
Yes, I have spun up a preview environment, and it works.